pkg/controllers/managementuser/controllers.go

```diff
@@ -38,9 +38,7 @@ func Register(ctx context.Context, mgmt *config.ScaledContext, cluster *config.U
 	windows.Register(ctx, clusterRec, cluster)
 	nsserviceaccount.Register(ctx, cluster)
 	if features.RKE2.Enabled() {
-		if err := snapshotbackpopulate.Register(ctx, cluster); err != nil {
-			return err
-		}
+		snapshotbackpopulate.Register(ctx, cluster)
```

Comment:
Would there be any value in logging the error?

Reply:
I don't think there are any errors that are potentially thrown downstream of the `Register` function that would bubble up to this point. We originally had the `err` allocation/check because we were creating a custom controller using a shared cache, but as we are not doing this anymore I removed this